wordle-web/pages/game.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-card class="mx-auto my-8" elevation="16" max-width="344">
+    <v-card-item>
+      <v-card-title> Hurdle Over This Wordle!!! </v-card-title>
+      <v-card-subtitle> <strong class="text-red-accent-4">hurdle it. I dare you. </strong> </v-card-subtitle>
+    </v-card-item>
+
+    <v-text-field
+      v-model="input"
+      label="Will you hurdle? (y/n):"
+      outlined
+      clearable />
+    <v-card-actions>
+      <v-spacer />
+      <v-btn
+        color="red-accent-4"
+        variant="elevated"
+        elevation="8"
+        @click="enterInput()">
+        Enter
+      </v-btn>
+    </v-card-actions>
+    <v-card-text>
+      {{ output }}
+    </v-card-text>
+    <v-card-text>
+      <NuxtLink to="/" external> Home page </NuxtLink>

```

Comment:
```suggestion
      <NuxtLink to="/"> Home page </NuxtLink>

```
'external` shouldn't be here since the homepage is still within the same app. [[Source]](https://nuxt.com/docs/api/components/nuxt-link#:~:text=Using-,external,-forces%20the%20link)

Reply:
Good catch! Fixed :)